### PR TITLE
Changed agent to support change from building/9 to building/10.

### PR DESCRIPTION
**Pull request by shdshdw**
This is required for the implementation of User story 1: As a housing corporation agent I want to have access to all percepts from the environment and all indicators that will be useful to me so that I can function in the best way possible

### CODE REVIEW

**Comment by lisv** [comment]
Are you sure you have covered all the buildings everywhere in the code?

File: src/nl/codefox/contextvh/constuct.test2g [hunk 1]
```diff
@@ -14,7 +14,7 @@ test construct(_) with
 		goal(constructBuilding(MultiPoly)) leadsto done(construct(MultiPoly)).
 		goal(constructBuilding(MultiPoly)) leadsto not(bel(emptyPoly(MultiPoly))).
 		goal(constructBuilding(MultiPoly)) leadsto bel(constructed(MultiPoly)).
-		goal(constructBuilding(MultiPoly)) leadsto bel(building(_, _, _, _, _, _, _, MultiPoly,_)).
+		goal(constructBuilding(MultiPoly)) leadsto bel(building(_, _, _, _, _, _, _, MultiPoly,_,_)).
```
Comment (MatthijsJWH) [inline src/nl/codefox/contextvh/constuct.test2g]: Please have a conistent style in your code.

**Comment by MatthijsJWH** [comment]
Also there are conflicts that have to be resolved before merging

**Comment by shdshdw** [comment]
I am attempting to fix merge conflicts now.

File: README.md [hunk 2]
```diff
@@ -73,9 +72,6 @@ ContextProject Virtual Humans for Serious Gaming
 [S6P]: Deliverables/Week%206/Backlog%20Sprint%206.pdf
 [S6R]: Deliverables/Week%207/Retrospective%20Sprint%206.pdf
 [S7P]: Deliverables/Week%207/Backlog%20Sprint%207.pdf
-[S7R]: Deliverables/Week%208/Retrospective%20Sprint%207.pdf
-[S8P]: Deliverables/Week%208/Backlog%20Sprint%208.pdf
```
Comment (JoshCode) [inline README.md]: This shouldn't be removed

File: README.md [hunk 3]
```diff
@@ -21,8 +21,7 @@ ContextProject Virtual Humans for Serious Gaming
 | 4      | [Sprint 4 Backlog][S4P] | [Sprint 4 Retrospective][S4R] | |
 | 5      | [Sprint 5 Backlog][S5P] | [Sprint 5 Retrospective][S5R] | [Architecture Design 2016.05.27][AD2016.05.27] |
 | 6      | [Sprint 6 Backlog][S6P] | [Sprint 6 Retrospective][S6R] | [Architecture Design 2016.06.03][AD2016.06.03] |
-| 7      | [Sprint 7 Backlog][S7P] | [Sprint 7 Retrospective][S7R] | [Architecture Design 2016.06.03][AD2016.06.10] |
-| 8      | [Sprint 8 Backlog][S8P] | Sprint 8 Retrospective (TBM) | |
```
Comment (JoshCode) [inline README.md]: This shouldn't be removed

File: README.md [hunk 4]
```diff
@@ -73,9 +72,6 @@ ContextProject Virtual Humans for Serious Gaming
 [S6P]: Deliverables/Week%206/Backlog%20Sprint%206.pdf
 [S6R]: Deliverables/Week%207/Retrospective%20Sprint%206.pdf
 [S7P]: Deliverables/Week%207/Backlog%20Sprint%207.pdf
-[S7R]: Deliverables/Week%208/Retrospective%20Sprint%207.pdf
-[S8P]: Deliverables/Week%208/Backlog%20Sprint%208.pdf
 [AD2016.05.06]: Deliverables/Week%203/Architecture_Design_2016.05.06.pdf
 [AD2016.05.27]: Deliverables/Week%206/Architecture_Design_2016.05.27.pdf
 [AD2016.06.03]: Deliverables/Week%207/Architecture_Design_2016.06.03.pdf
-[AD2016.06.10]: Deliverables/Week%208/Architecture_Design_2016.06.10.pdf
```
Comment (JoshCode) [inline README.md]: This shouldn't be removed

**Comment by shdshdw** [comment]
By the way, all the buildings have been checked in the code.

**Comment by MHMCD** [comment]
I have tested it and it still contains all buildings, works as intended, code looks good

Approved

**Comment by lisv** [comment]
I approve!

File: src/nl/codefox/contextvh/knowledgebase.pl [hunk 5]
```diff
@@ -14,9 +14,10 @@
 :- dynamic indicator/4.
 :- dynamic land/5.
 :- dynamic zone/5. 
-:- dynamic building/9.
+:- dynamic building/10.
 :- dynamic relevant_areas/2.
 
+
```
Comment (JoshCode) [inline src/nl/codefox/contextvh/knowledgebase.pl]: Useless new line

**Comment by MatthijsJWH** [comment]
I agree

**Comment by JoshCode** [comment]
It's fine now.

**Pull request merged**